index.js

```diff
@@ -151,5 +151,19 @@ if (!nativeBinding) {
 
 const { compress, decompress } = nativeBinding;
 
-module.exports.compress = compress;
-module.exports.decompress = decompress;
+// Error objects created via napi don't have JS stacks; wrap them so .stack is present
+// https://github.com/nodejs/node/issues/25318#issuecomment-451068073
+module.exports.compress = async function (data) {
```

Comment:
Note we will need to find another solution here. This file is generated by napi-rs so any `npm run build` is going to overwrite it.

Reply:
hmm interesting. (sorry for missing that!)

do you have a suggestion for how best to fix? a second wrapper file I guess is one option but doesn't feel satisfying

Reply:
I think the approach you implemented is probably the best way to go for now. Thank you!